internal/sdkv2provider/schema_cloudflare_zone_settings_override.go

```diff
@@ -138,6 +138,13 @@ var resourceCloudflareZoneSettingsSchema = map[string]*schema.Schema{
 		Computed:     true,
 	},
 
+	"speed_brain": {
```

Comment:
we only need to add this value to the latest schema (the others are migration schemas for going between versions)

Reply:
Okay so just to V1 , correct? I just updated.

Reply:
just to `resourceCloudflareZoneSettingsSchema` from memory. v0 and v1 are the transition states.